test/data/2022-09-09-cvOnDraftSaveBatched_request.json

```diff
@@ -0,0 +1,108 @@
+[
```

Comment:
could we keep name formats similar to ther test files, there is an index if the files represent similar req/resp


i would suggest going with `2022-09-09-cvOnDraftSave_2_request` for example, this will allow adding new versions of such files and keep the sorting in the explorer view in a way that all files within on group are together

Reply:
`test/data/2022-09-09-cvOnDraftSaveBatched_2_request.json` seems like a reasonable extension of this name. I couldn't tell by eyeballing #825 if we have other batched responses present.

Ideally, if JSON had comments in it, I'd put the information into the file instead of just the file name.